sqe-http-api/DTOs/User.cs

```diff
@@ -22,4 +22,54 @@ public class LoginResponseDTO : UserDTO
     {
         public string token { get; set; }
     }
+    
+    public class NewUserDTO
```

Comment:
Why is this different from the regular UserDTO?

Reply:
Renamed according to comment below.

Done.